plugins/org.jkiss.dbeaver.ext.mssql/src/org/jkiss/dbeaver/ext/mssql/SQLServerUtils.java

```diff
@@ -342,6 +341,30 @@ public static String changeCreateToAlterDDL(
         return ddl;
     }
 
+
+    @NotNull
+    public static String getObjectDefinitionFunction(
+        @NotNull String systemSchema,
```

Comment:
Don't pass it as a parameter; retrieve it via `SQLServerUtils#getSystemSchemaFQN` right here. The caller's parameter is also not used anywhere else.